cadence/values.yaml

```diff
@@ -57,6 +57,65 @@ server:
   affinity: {}
 
   config:
+    # clusterMetadata contains the settings of the Cadence clusters
+    # participating in a Cadence cluster group.
+    clusterMetadata:
+      # enableGlobalDomains when set to true allows creating global domains
+      # which are replicated between all clusters and whose tasks are served by
+      # the active cluster in the replication.
+      #
+      # WARNING: This value **MUST NOT** be changed after launching the Cadence
+      # cluster group.
+      enableGlobalDomain: true
+
+      # failoverVersionIncrement is the increment of each cluster version when
+      # failover happens.
+      #
+      # WARNING: For each cluster, (initialFailoverVersion +
+      # i*failoverVersionIncrement) **MUST** result in a unique value.
+      failoverVersionIncrement: 10
+
+      # masterClusterName is the name of the only cluster which can register
+      # and update a domain, although each cluster can do domain failover.
+      #
+      # WARNING: This value **MUST NOT** be changed after launching the Cadence
+      # cluster group.
+      masterClusterName: "cluster-0"
```

Comment:
Also, use `primaryClusterName`. This is being deprecated https://github.com/uber/cadence/blob/9a46d9d483cfdb8fd5ca77d1f448d79d78b0e791/common/config/config.go#L296
Though it's still supported for compatibility. But it will be removed eventually.

Reply:
I tried using that, but it caused a panic using server 0.21.3, so I opted for using `masterClusterName` instead which worked.

```
-> kubectl --kubeconfig "${BANZAI_INSTALLER_WORKSPACE_KUBECONFIG}" --namespace banzaicloud logs cadence-worker-67d87b5876-b756t

# Begin: 2021-08-02T11:14:30.679566000+02:00

+ CONFIG_TEMPLATE_PATH=/etc/cadence/config/config_template.yaml
+ dockerize -template /etc/cadence/config/config_template.yaml:/etc/cadence/config/docker.yaml
+ exec cadence-server --root /etc/cadence --env docker start --services=worker
2021/08/02 09:14:19 Loading config; env=docker,zone=,configDir=/etc/cadence/config
2021/08/02 09:14:19 Loading configFiles=[/etc/cadence/config/docker.yaml]
{"level":"info","ts":"2021-08-02T09:14:19.027Z","msg":"Updated dynamic config","service":"cadence-worker","logging-call-at":"file_based_client.go:258"}
{"level":"info","ts":"2021-08-02T09:14:19.028Z","msg":"Listening for TChannel requests","service":"cadence-worker","address":"10.244.0.35:7939","logging-call-at":"rpc.go:90"}
panic: Master cluster name is empty

goroutine 1 [running]:
github.com/uber/cadence/common/cluster.NewMetadata(0x332ee00, 0xc000104eb0, 0xc0006dd0c0, 0xa, 0x0, 0x0, 0xc0003c0350, 0x9, 0xc000448ea0, 0xc0005f72a8, ...)
        /cadence/common/cluster/metadata.go:88 +0x683
github.com/uber/cadence/cmd/server/cadence.(*server).startService(0xc0005745d0, 0xc000448a50, 0x7ffe6d7be009)
        /cadence/cmd/server/cadence/server.go:145 +0x8f0
github.com/uber/cadence/cmd/server/cadence.(*server).Start(0xc0005745d0)
        /cadence/cmd/server/cadence/server.go:81 +0x6c
github.com/uber/cadence/cmd/server/cadence.startHandler(0xc0000d0160)
        /cadence/cmd/server/cadence/cadence.go:83 +0x79c
github.com/uber/cadence/cmd/server/cadence.BuildCLI.func1(0xc0000d0160)
        /cadence/cmd/server/cadence/cadence.go:208 +0x2b
github.com/urfave/cli.HandleAction(0x2655c60, 0x2d009b8, 0xc0000d0160, 0xc0000d0160, 0x0)
        /go/pkg/mod/github.com/urfave/cli@v1.22.4/app.go:528 +0x127
github.com/urfave/cli.Command.Run(0x2bfad0d, 0x5, 0x0, 0x0, 0xc000459fc0, 0x1, 0x1, 0x2c179c0, 0x14, 0x0, ...)
        /go/pkg/mod/github.com/urfave/cli@v1.22.4/command.go:174 +0x528
github.com/urfave/cli.(*App).Run(0xc000413340, 0xc00004c070, 0x7, 0x7, 0x0, 0x0)
        /go/pkg/mod/github.com/urfave/cli@v1.22.4/app.go:279 +0x731
main.main()
        /cadence/cmd/server/main.go:36 +0x83

# End: 2021-08-02T11:14:30.855135000+02:00
# Duration: 175ms
# Exit code: 0
```

Reply:
Ahh you are right. It's not released yet. 
https://github.com/uber/cadence/blob/v0.21.3/common/config/config.go

And good point about compatibility. Even if it's released, we still need to keep using masterClusterName for older versions.